org.adempiere.base/src/org/compiere/acct/Doc_AllocationHdr.java

```diff
@@ -190,6 +190,7 @@ public ArrayList<Fact> createFacts (MAcctSchema as)
 		Fact factForRGL = new Fact(this, as, Fact.POST_Actual); // dummy fact (not posted) to calculate Realized Gain & Loss
 		boolean isInterOrg = isInterOrg(as);
 		MAccount bpAcct = null;		//	Liability/Receivables
+		ArrayList<Integer> bpAccts = new ArrayList<Integer>();
```

Comment:
it look like we has many bpAcct as many invoice but actual we has 2 bpAcct for ar and ap only